ekncontent/docs/reference/ekncontent/eos-knowledge-content-sections.txt

```diff
@@ -80,7 +80,9 @@ eknc_get_running_under_flatpak
 EkncContentObjectError
 <SUBSECTION Standard>
 EKNC_CONTENT_OBJECT_ERROR
+EKNC_TYPE_CONTENT_OBJECT_ERROR
 <SUBSECTION Private>
+eknc_content_object_error_get_type
```

Comment:
This seems like a mistake, I think usually the error code enums don't get registered as enum types